### DISCUSSION TITLE
Simpler, more minimal examples

### DISCUSSION THREAD

**Discussion by MoSattler** [wix/react-native-ui-lib#1824] [Ideas]
Hi!

I was wondering if you guys think there could be any value added by showing simpler examples. 
Some of them, while probably very well coded and easy to read for somebody familiar with the code, are not so easy to understand if you just want to know how a component works. 

E.g. I wanted to get a very minimal Drawer, and the documentation linked me [here](https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/DrawerScreen.tsx) to check out an example.

As somebody not to familiar with the code, it was not exactly easy to understand what is going on. There are multiple imports from other files, and quite a few function calls that are not trivial. 

If you guys think that more minimal, easy to follow examples would be desirable, I'd be happy to contribute!

**Comment by ethanshar** [comment]
Hi, @MoSattler 
This is definitely something we're working on. 
Soon our docs will also present s quick snippet you'll be able to copy and use in your code with no much effort.

**Comment by ethanshar** [comment]
Hi, 
I'm happy to update our docs now include quick code snippets for our components! 🎉 
If you find a specific snippet not clear enough, please open an issue and we'll fix it.

**Comment by MoSattler** [comment]
@ethanshar sounds good, thank you so much!

**Comment by MoSattler** [comment]
Hi @ethanshar, I stumbled on quite a few code examples which were wrong, like [this](https://wix.github.io/react-native-ui-lib/docs/components/overlays/FloatingButton#usage) or [this](https://wix.github.io/react-native-ui-lib/docs/components/basic/Chip#usage) one.
Where can I open some PRs to fix them?
